lib/toTermArray.js

```diff
@@ -2,7 +2,7 @@ const term = require('./term')
 const toArray = require('./toArray')
 
 function toTermArray (items, type, languageOrDatatype) {
-  if (!items && !type) {
+  if (items !== false && !items && !type) {
```

Comment:
Related, `!items` also creates a problem with `0` (refs https://github.com/libero/article-store/blob/c4f460169e4a58159a7b99ec0e52ee2a7c83e15d/src/routes/article-list.ts#L30).

Should the check be like https://github.com/rdf-ext/clownface/pull/18?